src/status_im2/contexts/wallet/common/collectibles_tab/view.cljs

```diff
@@ -1,16 +1,22 @@
 (ns status-im2.contexts.wallet.common.collectibles-tab.view
   (:require
+    [clojure.string :as string]
     [quo.core :as quo]
     [react-native.core :as rn]
     [status-im2.contexts.wallet.common.empty-tab.view :as empty-tab]
-    [status-im2.contexts.wallet.common.temp :as temp]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(defn- collectible-image-url
```

Comment:
should we not put this in the sub directly?

Reply:
Updated the code to add the result of this function as an additional key to the collectible map.